src/components/layout/useDeviceSizes.jsx

```diff
@@ -0,0 +1,26 @@
+import { useEffect, useState } from "react";
+
+function getWindowDimensions() {
```

Comment:
delete this file

Reply:
Umm, I was using this to set the number of product in preview of searchBox. In `full width`; 3 product will be filtered and shown, in `tablet size`; 2 product, in `smaller screen`; none. But I can try `display: none` in media queries if you say so.

Reply:
I changed the way into media queries. (thanks)